src/AbstractActionHandler.ts

```diff
@@ -17,9 +17,9 @@ import {
  *
  */
 export abstract class AbstractActionHandler {
-  protected lastProcessedBlockNumber: number = 0
-  protected lastProcessedBlockHash: string = ""
-  protected handlerVersionName: string = "v1"
+  public lastProcessedBlockNumber: number = 0
```

Comment:
we could publicly expose these or add a method explicitly stating what we're trying to get.

e.g.
```
public getActionHandlerStatus(): ActionHandlerStatus() {
 return {...} 
}
```